health/notifications/alarm-notify.sh.in

```diff
@@ -391,6 +391,7 @@ STACKPULSE_WEBHOOK=
 
 # opsgenie configs
 OPSGENIE_API_KEY=
+OPSGENIE_API_URL=${OPSGENIE_API_URL:-"https://api.opsgenie.com"}
```

Comment:
this line should be after

https://github.com/tinyhammers/netdata/blob/db489de43336d8c2bd576ee24124fd4053ed7018/health/notifications/alarm-notify.sh.in#L396-L413

Reply:
@thiagoftsm that should fix the issue you mentioned ^^

Reply:
This is not happening @ilyam8 :

```bash
bash-5.1# grep "https://api.opsgenie.com" alarm-notify.sh 
OPSGENIE_API_URL=${OPSGENIE_API_URL:-"https://api.opsgenie.com"}
bash-5.1# ./alarm-notify.sh test

# SENDING TEST WARNING ALARM TO ROLE: sysadmin
2021-02-01 16:48:21: alarm-notify.sh: INFO: sent slack notification for: hades test.chart.test_alarm is WARNING to '#myalarms'
2021-02-01 16:48:22: alarm-notify.sh: INFO: sent telegram notification for: hades test.chart.test_alarm is WARNING to '-363762934'
curl: (3) URL using bad/illegal format or missing URL
2021-02-01 16:48:22: alarm-notify.sh: ERROR: failed to send opsgenie notification for: hades test.chart.test_alarm is WARNING, with HTTP error code 000.

```

Reply:
I ran now another test with master branch:

```bash
bash-5.1# grep "https://api.opsgenie.com" alarm-notify.sh 
  httpcode=$(docurl -X POST -H "Content-Type: application/json" -d "${payload}" "https://api.opsgenie.com/v1/json/integrations/webhooks/netdata?apiKey=${OPSGENIE_API_KEY}")
bash-5.1# ./alarm-notify.sh test

# SENDING TEST WARNING ALARM TO ROLE: sysadmin
2021-02-01 17:05:39: alarm-notify.sh: INFO: sent slack notification for: hades test.chart.test_alarm is WARNING to '#myalarms'
2021-02-01 17:05:40: alarm-notify.sh: INFO: sent telegram notification for: hades test.chart.test_alarm is WARNING to '-99999'
2021-02-01 17:05:40: alarm-notify.sh: INFO: sent opsgenie notification for: hades test.chart.test_alarm is WARNING
# OK
```

Everything worked, so the PR needs to be fixed.

Reply:
> This is not happening @ilyam8 :

I said that we need to put `OPSGENIE_API_URL=${OPSGENIE_API_URL:-"https://api.opsgenie.com"}` after `health_alarm_notify.conf` loading. Did you try it?

---

Take attention to the line numbers

```cmd
[ilyam@pc plugins.d]$ grep -n "https://api.opsgenie.com" alarm-notify.sh
394:#OPSGENIE_API_URL=${OPSGENIE_API_URL:-"https://api.opsgenie.com"}
415:OPSGENIE_API_URL=${OPSGENIE_API_URL:-"https://api.opsgenie.com"}
```

```cmd
[ilyam@pc plugins.d]$ ./alarm-test.sh

# SENDING TEST WARNING ALARM TO ROLE: sysadmin
2021-02-02 11:55:27: alarm-notify.sh: WARNING: Cannot find file '/opt/netdata/etc/netdata/health_alarm_notify.conf'.
2021-02-02 11:55:27: alarm-notify.sh: ERROR: failed to send opsgenie notification for: pc test.chart.test_alarm is WARNING, with HTTP error code 400.
# FAILED

# SENDING TEST CRITICAL ALARM TO ROLE: sysadmin
2021-02-02 11:55:27: alarm-notify.sh: WARNING: Cannot find file '/opt/netdata/etc/netdata/health_alarm_notify.conf'.
2021-02-02 11:55:28: alarm-notify.sh: ERROR: failed to send opsgenie notification for: pc test.chart.test_alarm is CRITICAL, with HTTP error code 400.
# FAILED

# SENDING TEST CLEAR ALARM TO ROLE: sysadmin
2021-02-02 11:55:28: alarm-notify.sh: WARNING: Cannot find file '/opt/netdata/etc/netdata/health_alarm_notify.conf'.
2021-02-02 11:55:28: alarm-notify.sh: ERROR: failed to send opsgenie notification for: pc test.chart.test_alarm is CLEAR, with HTTP error code 400.
# FAILED
```

Reply:
@ilyam8 I tested now and when we move it works.

@tinyhammers , please, move the variable declaration for the line 415 a @ilyam8 described in the previous comment.